### PR TITLE
Move pkg/util/netutils to mostly pkg/network/common

### PR DESCRIPTION
**Pull request by damemi**
Most of these functions were only used in pkg/network with few callers, so this moves them there

### CODE REVIEW

File: pkg/oc/cli/admin/prune/imageprune/helper.go [hunk 1]
```diff
@@ -175,7 +175,7 @@ func TryProtocolsWithRegistryURL(registry string, allowInsecure bool, action fun
 	switch {
 	case len(url.Scheme) > 0 && url.Scheme != "unset":
 		protos = []string{url.Scheme}
-	case allowInsecure || netutils.IsPrivateAddress(registry):
+	case allowInsecure || common.IsPrivateAddress(registry):
```
Comment (deads2k) [inline pkg/oc/cli/admin/prune/imageprune/helper.go]: naughty.  no

File: pkg/network/common/common.go [hunk 2]
```diff
@@ -168,3 +168,105 @@ func GetNetworkInfo(networkClient networkclient.Interface) (*NetworkInfo, error)
...
+	}
+	return net, nil
+}
+
+var localHosts []string = []string{"127.0.0.1", "::1", "localhost"}
+var localSubnets []string = []string{"10.0.0.0/8", "172.16.0.0/12", "192.168.0.0/16", "fc00::/7", "fe80::/10"}
+
+// IsPrivateAddress returns true if given address in format "<host>[:<port>]" is a localhost or an ip from
+// private network range (e.g. 172.30.0.1, 192.168.0.1).
+func IsPrivateAddress(addr string) bool {
```
Comment (deads2k) [inline pkg/network/common/common.go]: This function will get promote to library-go. `pkg/network/networkutils`

Comment (damemi) [reply pkg/network/common/common.go]: Moved https://github.com/openshift/library-go/pull/422

**Pull request merged**